README.md

```diff
@@ -1 +1,6 @@
-# Hello-Microverse-
\ No newline at end of file
+Major Languages used are 
+Html
+Frmeworks used are 
+Css
+Techonolegies used
+Git and github
```

Comment:
Part of the requirements of the project is to create a descriptive README file as the last code reviewer suggested. Kindly update your README file using the template provided in this [link](https://github.com/microverseinc/readme-template). Please don't forget to customize it. :+1: :smiley: :smiley: